spec/api/labels_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Labels API.' do
+  let!(:account) { create :account }
+  let!(:label) { create :label, account: account }
+
+  let!(:access_token) { create :doorkeeper_token, resource_owner_id: account.id }
```

Comment:
you should use doorkeeper shared context here.
Example https://github.com/rubykube/barong/blob/87aa2a2690236eb71ed2c48ced48fe02f0878333/spec/api/v1/accounts_spec.rb#L4-L7

Reply:
thx, done